apps/user_status/lib/Controller/PredefinedStatusController.php

```diff
@@ -57,12 +60,12 @@ public function __construct(string $appName,
 	/**
 	 * @NoAdminRequired
 	 *
-	 * @return DataResponse
+	 * @return DataResponse<PredefinedStatus[]> 200
 	 */
-	public function findAll():DataResponse {
+	public function findAll(): DataResponse {
 		// Filtering out the invisible one, that should only be set by API
-		return new DataResponse(array_filter($this->predefinedStatusService->getDefaultStatuses(), function (array $status) {
-			return !array_key_exists('visible', $status) || $status['visible'] === true;
+		return new DataResponse(array_filter($this->predefinedStatusService->getDefaultStatuses(), function (PredefinedStatus $status) {
+			return $status->isVisible();
 		}));
```

Comment:
```suggestion
		return new DataResponse(array_filter($this->predefinedStatusService->getDefaultStatuses(), fn (PredefinedStatus $status) => $status->isVisible()));
```
In this instance using the `fn` keyword is improving readability I think.

Reply:
Didn't know that existed!